### ISSUE TITLE
Multiple InCHlib objects on the same page

### ISSUE THREAD

**Issue by JEFworks** [skutac/InCHlib.js#9]
Hi,

I would like to display multiple InCHlib heatmaps on the same page that interact with each other; for example, clicking on the row of the main InCHlib heatmap would trigger drawing a second InCHlib heatmap. But currently, due to the way the global "_this" and "this" objects are being used, this is not possible. What is the best way to go about having multiple InCHlib heatmaps on the same page?

Thanks in advance for your help and insights!

Best,
Jean

**Comment by skutac** [comment]
Hi Jean,
thank you for this bug report.

I tested this functionality long time ago and it worked. But then I changed the local instancing to one global and I didn't realize that it would make impossible to use multiple InCHlib instances on a single web page.

Anyway, I've returned it to the original state (local instancing) so it should work now. ;-)

As always this issue is fixed only in the dev version for now. I hope to release new version (with examples etc.) in two weeks.

Best,
Ctibor
